### PR TITLE
fix: update dependabot config

### PR DESCRIPTION
**Pull request by hanabi1224**
**Summary of changes**

To address https://github.com/ChainSafe/forest/pull/3561#pullrequestreview-1666507576

Changes introduced in this pull request:
- ignore fvm related updates (we have to manually make the updates in most cases)
- bump patch versions with a single PR

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -8,6 +8,18 @@ updates:
     open-pull-requests-limit: 1
     schedule:
       interval: "weekly"
+    # https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#example-disabling-version-updates-for-some-dependencies
+    ignore:
+      - dependency-name: "cid"
+      - dependency-name: "*ipld*"
+      - dependency-name: "bls-signatures"
+      - dependency-name: "fvm*"
+      - dependency-name: "filecoin*"
```
Comment (LesnyRumcajs) [inline .github/dependabot.yml:17]: Do we want to ignore patches as well?

Comment (hanabi1224) [reply .github/dependabot.yml]: Fixed

File: .github/dependabot.yml [hunk 2]
```diff
@@ -8,6 +8,28 @@ updates:
     open-pull-requests-limit: 1
     schedule:
       interval: "weekly"
+    # https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#example-disabling-version-updates-for-some-dependencies
```
Comment (LesnyRumcajs) [inline .github/dependabot.yml:11]: Why do we want to discard minor updates, e.g., `cid` or `bls-signature`?

Comment (hanabi1224) [reply .github/dependabot.yml:11]: `cid` is dependency of `fvm*` crates, minor version updates make them incompatible

e.g.
```toml
[[package]]
name = "fvm_ipld_blockstore"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "417f52f6915b9f9a68de8462e1cf46f14a2c16420f484b8d2066873de2ffe420"
dependencies = [
 "anyhow",
 "cid",
 "multihash",
]

[[package]]
name = "fvm_ipld_encoding"
version = "0.4.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "90608092e31d9a06236268c58f7c36668ab4b2a48afafe3a97e08f094ad7ae50"
dependencies = [
 "anyhow",
 "cid",
 "fvm_ipld_blockstore",
 "multihash",
 "serde",
 "serde_ipld_dagcbor",
 "serde_repr",
 "serde_tuple",
 "thiserror",
]
```

**Review by LesnyRumcajs — APPROVED** [review]

**Review by lemmih — APPROVED** [review]

**Pull request merged**